### PR TITLE
Document town map

### PR DESCRIPTION
**Pull request by gensarde**

### CODE REVIEW

File: include/overlay080/struct_town_map_city_holder.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#ifndef POKEPLATINUM_STRUCT_TOWN_MAP_CITY_HOLDER_H
+#define POKEPLATINUM_STRUCT_TOWN_MAP_CITY_HOLDER_H
+
+#include "overlay080/struct_town_map_city.h"
+
+typedef struct TownMapCityHolder {
```
Comment (lhearachel) [inline include/overlay080/struct_town_map_city_holder.h:6]: Is this just the full map struct with the blocks arranged?

Comment (Nomura-RH) [reply include/overlay080/struct_town_map_city_holder.h:6]: > Is this just the full map struct with the blocks arranged?

Nope, it's just a collection of blocks (actually cities, which are blocks + some other data)

Comment (lhearachel) [reply include/overlay080/struct_town_map_city_holder.h:6]: Right, but what I'm curious about is what it represents. Is it all the blocks that go on the map? Is it a set of blocks for those multi-block cities (like how Jubilife or Hearthome are 2x2 squares, whereas Sandgem is a single block)?

Comment (Nomura-RH) [reply include/overlay080/struct_town_map_city_holder.h:6]: > Right, but what I'm curious about is what it represents. Is it all the blocks that go on the map? Is it a set of blocks for those multi-block cities (like how Jubilife or Hearthome are 2x2 squares, whereas Sandgem is a single block)?

Yes, it holds the data for all the city (plus Pal Park and Victory Road) blocks that go on the map. There are various kind of blocks: 1x1 (like Sandgem), 2x2 (like Jubilife), L-shaped and Pal-Park style (green circle). Each city -> 1 block.

**Comment by lhearachel** [comment]
Closing as abandoned

**Pull request closed without merging**